### DISCUSSION TITLE
External .hidden class interfering with my element

### DISCUSSION THREAD

**Discussion by hexpit** [tailwindlabs/tailwindcss#5860] [Help]
In my project I use editor.js to create a blog. Editor.js soucre spits some incline css with 

```
.hidden {
  display: none;
}
```

This makes some of the responsive elements on my page go disapper for example:
`<div class="ml-auto flex sm:block hidden">`

how to overcome that issue without editing source files of Editor.js?

**Comment by DuddiD** [comment]
May I suggest that you prefix your tailwind classes? If you are using a framework or third party plugin, some classes might be the same that tailwind uses. To prevent this, and make tailwind classes unique, add a prefix in your tailwind.config.js file like this:

```
module.exports = {
  prefix: 'tw-',
  // ...
}
```

Now doing this you have to remember that all classes you add to your HTML has to have your prefix, so your classes would now look like this:

`<div class="tw-ml-auto tw-flex sm:tw-block tw-hidden">`

Note that you don't need the prefix on the breakpoint name. Hope this helps.

**Reply by hexpit** [reply]
@DuddiD I appreciate your suggestion, but it will be really cumborsome to append prefix on almost completed project just for few pages?

**Comment by NovusOpus** [comment]
Take a look at : [Selector strategy](https://tailwindcss.com/docs/configuration#selector-strategy), Maybe this can help you work around your issue.
